content/docs/dvclive/api-reference/live/index.md

```diff
@@ -51,7 +47,7 @@ files.
   `path` directory and start from that point. _Default_: `False`.
 
 - `summary` - If `True`, upon each `Live.log()` call, DVCLive will generate a
-  _metrics summary_ (usable by `dvc metrics`). The _summary_ will be located at
+  summary (usable by `dvc metrics`). The summary will be located at
   `{path}.json`. _Default_: `True`.
```

Comment:
What do you think about a warnings to not change `resume` or `summary` in no-step scenarios?

Reply:
Added warning. TBH I would remove `summary` as I don't think it'd be much trouble for users if we make the summary mandatory